src/app/pages/camper/schedule/schedule.component.scss

```diff
@@ -0,0 +1,47 @@
+$grey: #ABABAB;
+$red: #B7374A;
+
+td:nth-child(1){
+	width: 130px;
+}
+
+#schedule{
+	margin-top:60px;
+}
+#schedule li:nth-child(1){
+	border:solid 1px $grey;
```

Comment:
after the attribute name and colon, there should be a space 

i.e. 
border: solid 1px $grey 

instead of 
border:solid 1px $grey